src/dlgTriggerEditor.cpp

```diff
@@ -4738,56 +4799,168 @@ void dlgTriggerEditor::slot_var_selected(QTreeWidgetItem *pItem)
 
 void dlgTriggerEditor::slot_action_selected(QTreeWidgetItem *pItem)
 {
-    if( ! pItem ) return;
+    if( ! pItem )
+    {
+        return;
+    }
 
     // save the current action before switching to the new one
     if ( pItem != mpCurrentActionItem )
+    {
         saveAction();
+    }
 
-    mpActionsMainArea->show();
-    mpSourceEditor->show();
-
-    mpSystemMessageArea->hide();
-    mpSourceEditor->setPlainText( "" );
-
-    mpActionsMainArea->lineEdit_action_icon->clear();
-    mpActionsMainArea->lineEdit_action_name->clear();
-    mpActionsMainArea->checkBox_pushdownbutton->setChecked( false );
-    mpActionsMainArea->buttonColumns->clear();
-    mpActionsMainArea->css->clear();
+    // As there is more than one place where this is needed save it as a variable
+    QString normalNameToolTipText = QStringLiteral( "<html><head/><body><p>%1</p></body></html>" )
+                                    .arg( tr( "Enter a unique name for this item, future code will allow some control of permanent "
```

Comment:
Perhaps we ought to explain somewhere why unique names are normally advisable - ideally the script-able Lua functions that act on items that take a name ought to also take an ID _number_ instead to work on a particular one - though getting around the case where the name is a number and avoiding Lua coercion of strings <==> numbers could be fun!